shared/locales/en/translation.json

```
@@ -198,6 +198,11 @@
   ],
   "homeHeroGradientSubtext": "From concept to code, we craft human-centered GenAI experiences.",
   "homeHeroSubtext": "From concept to code, we craft human-centered GenAI experiences.",
```

Biome (2.1.2)

[error] The key homeHeroSubtext was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)